web/src/components/pages/dashboard/dashboard.css

```diff
@@ -174,3 +173,123 @@
         }
     }
 }
+.challenge-bar {
+    width: 451px;
+    padding: 0 20px;
+    display: flex;
+    justify-content: space-between;
+}
+.points {
+    width: 321px;
+    height: 40px;
+    border-radius: 19px;
+    border: solid 1px var(--light-grey);
+    background-color: var(--white);
+    display: flex;
+    line-height: 1.71;
+    align-items: center;
+    justify-content: center;
+    padding: 0 15px;
+    box-sizing: border-box;
+    img {
+        width: 14px;
+        height: 14px;
+    }
+    span {
+        margin: 0 auto;
+    }
+    .score {
+        font-weight: 600;
+    }
+    > .label {
```

Comment:
I'd like to keep our selectors as simple as possible. Can we remove the child combinator (`>`) from all of these styles, or is it needed for some reason?

Reply:
Thanks, @rileyjshaw.
Your suggestion is helpful. But I think these selectors' name is too general. If import other styles, I concern about the same name selectors existed. The styles will be overwritten.

Reply:
I agree that `.label` would be too general, but since you’ve nested it, the selector will already resolve to `.points .label`. My comment was on the child combinator (`>`); I think it’s overkill to set the selector as `.points > .label`. Removing the `>` should still be specific enough, and makes the selector more resilient to changes in the component structure.

It’s just a small nitpick, certainly not a blocker. I'll land your changes now, thanks for the awesome work!

Reply:
You are right. I will pay attention to it in my future work.